src/main/java/ca/fxco/configurablepistons/blocks/pistons/basePiston/BasicMovingBlockEntity.java

```diff
@@ -307,6 +293,9 @@ public void finalTick() {
             if (stick.usesConfigurablePistonStickiness() && stick.isSticky(this.movedState)) {
                 this.finalTickStuckNeighbors(stick.stickySides(this.movedState));
             }
+
+            this.progress = 1.0F;
```

Comment:
I don't think it'll cause issues, but for consistency's sake this should maybe be moved back up to before the `this.finishMovement()` call.

Reply:
If we do that, we break the `finalTickStuckNeighbors`

Reply:
I can get around this although its not the best solution